app/client/survey_admin/survey_admin_details.coffee

```diff
@@ -20,26 +21,29 @@ Template.survey_admin_details.onCreated ->
               formQuestions.push questionProps
             formProps.questions = formQuestions
             instance.forms.insert formProps
+            instance.active.set true
```

Comment:
Good catch. We should set `active` to the value of the property on the Parse object.
When we create surveys we need to set the active state to false.
I'll take care of these changes.